tests/scripts/thread-cert/Cert_8_3_01_CommissionerPetition.py

```diff
@@ -323,8 +323,8 @@ def verify(self, pv):
                              {
                               NM_COMMISSIONER_SESSION_ID_TLV
                              } == set(p.thread_meshcop.tlv.type) and\
-                   p.mle.tlv.leader_data.data_version >=
-                   (_dr_pkt2.mle.tlv.leader_data.data_version + 1) % 256 and\
+                   (p.mle.tlv.leader_data.data_version -
```

Comment:
It seems the network data version changed twice when stopping the commissioner - 1) commissioner sends c/cs to clear steering data when clearing joiners
https://github.com/openthread/openthread/blob/3166f6a0ed9cd7a5aa4ecea90605ef39dd5547bf/src/core/meshcop/commissioner.cpp#L421-L429
2) commissioner sends c/la to resign

I am thinking 1) could be saved to avoid frequent network data change as Leader will reset commissioning data when resigning commissioner after receiving 2)

Thoughts? @simonlingoogle @jwhui